code/game/machinery/firealarm.dm

```diff
@@ -57,32 +57,35 @@
 
 /obj/machinery/firealarm/update_icon()
 	cut_overlays()
-
+	SSvis_overlays.remove_vis_overlay(src, managed_vis_overlays)

```

Comment:
@ninjanomnom is this corrrect?

Reply:
This is the same stuff as used on air alarms, computers, and APCs